README.md

```diff
@@ -10,8 +11,20 @@ The sample solution has the following requirements:
 * Build Tool: The solution requires Visual studio or .NET Core CLI to be installed to be able to run the sample projects.
 
 ## Authentication Setup
-The credentials file and corresponding private key file for the samples is ```pdfservices-api-credentials.json``` and ```private.key``` respectively.
-Before the samples can be run, replace both the files with the ones present in the downloaded zip file at the end of creation of credentials via [Get Started](https://www.adobe.io/apis/documentcloud/dcsdk/gettingstarted.html?ref=getStartedWithServicesSdk) workflow.
+The credentials file for the samples is ```pdfservices-api-credentials.json```.
+Before the samples can be run, set the environment variables `CLIENT_ID` and `CLIENT_SECRET` from the `pdfservices-api-credentials.json` file downloaded at the end of creation of credentials via [Get Started](https://www.adobe.io/apis/documentcloud/dcsdk/gettingstarted.html?ref=getStartedWithServicesSdk) workflow by running the following commands :
```

Comment:
Let's rename to PDF_SERVICES_CLIENT_ID and PDF_SERVICES_CLIENT_SECRET

Reply:
Done